features/support/fixtures.rb

```diff
@@ -97,6 +97,7 @@ def self.digital_specialists_brief
       title: "Tea drinker",
       workingArrangements: "Hard work",
       workplaceAddress: "London",
+      questionAndAnswerSessionDetails: "Here be Q&A details"
```

Comment:
Wondering if we should add this for all lots (i.e. outcomes and URP too)? Thoughts?